src/components/App.js

```diff
@@ -1 +1,24 @@
-// create your App component here
+import React, { useState, useEffect } from "react"
+
+const url = "https://dog.ceo/api/breeds/image/random"
+
+function App() {
+  const [getDogPic, setGetDogPic] = useState(null)
```

Comment:
You wouldn't need to call the `getDogPic`. `dogPic` would be a better name since there's no fetching or data retrieval happening for the variable.
```suggestion
  const [dogPic, setGetDogPic] = useState(null)
```